pkg/monitortests/kubeapiserver/incluster_disruption_serializer/manifests/namespace.yaml

```diff
@@ -1,7 +1,7 @@
 kind: Namespace
 apiVersion: v1
 metadata:
-  generateName: e2e-disruption-monitor-
+  name: e2e-disruption-monitor
```

Comment:
This is a fix in the wrong direction.  YOu can either add RBAC for the SCC or set the runlevel=0 to skip the SCC check in this namespace.

Reply:
I did add RBAC for SCC, but for some reason we run disruption tests twice (once during uprgade and once during conformance tests). Its unreasonable to create a new namespace for the second run

Reply:
> Its unreasonable to create a new namespace and new RBAC rules for the second run

On what basis?

New namespaces prevent overlap and allow multiple watchers.  Collapsing to a single namespace is an anti-goal.  I'm ok with RBAC if that's what you prefer.  I'm also ok with runlevel=0.

Reply:
There is no reason to run multiple disruption watchers for the same apiservers in parallel